include/FilesManager.h

```diff
@@ -29,4 +37,13 @@ class FilesManager {
 	 * Loads all data from space_objects_data.csv file into a private vector
 	 */
 	void loadSpaceObjectsData();
+
+	/**
+	 * @brief
+	 * Saves simulation data into the binary file and destroys the object
+	 * 
+	 * @param simulation simulation object containing all the simulation data
+	 * @param fileName a fileName to save without any prefix or the extension
+	 */
+	void saveSimulationData(Simulation* simulation, string fileName, string dirname = "dane", string extension = ".bin");
```

Comment:
Tu myślę, że można w tym data zapisywać, ale jeżeli chcemy dodać to w innym folderze to trzeba w cmakelist.txt dopisać COPY do tego folderu
```suggestion
	void saveSimulationData(Simulation* simulation, std::string fileName, std::string dirname = "data", std::string extension = ".bin");
```